setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = inductiva
-version = 0.0.8
+version = 0.0.14
```

Comment:
Why did change the version? Are we updating versions every time we push something?

Reply:
No, only everytime it goes to PyPi.